tests/functional/preview_and_dev/test_cbc_integration.py

```diff
@@ -67,11 +42,10 @@ def test_get_loopback_request_with_bad_id_returns_no_items():
     assert len(response["Items"]) == 0
 
 
-@pytest.mark.skip("Skip while vodafone loopback architecture issue is investigated")
+# @pytest.mark.skip("Skip while vodafone loopback architecture issue is investigated")
```

Comment:
If we no longer need to skip we should remove this

Reply:
comment removed